edge_computing/cnf-talm-for-cluster-upgrades.adoc

```diff
@@ -39,11 +39,11 @@ include::modules/cnf-topology-aware-lifecycle-manager-about-subscription-crs.ado
 
 include::modules/cnf-topology-aware-lifecycle-manager-apply-policies.adoc[leveloffset=+2]
 
-include::modules/cnf-topology-aware-lifecycle-manager-backup-concept.adoc[leveloffset=+1]
+//include::modules/cnf-topology-aware-lifecycle-manager-backup-concept.adoc[leveloffset=+1]
 
-include::modules/cnf-topology-aware-lifecycle-manager-backup-feature.adoc[leveloffset=+2]
+//include::modules/cnf-topology-aware-lifecycle-manager-backup-feature.adoc[leveloffset=+2]
 
-include::modules/cnf-topology-aware-lifecycle-manager-backup-recovery.adoc[leveloffset=+2]
+//include::modules/cnf-topology-aware-lifecycle-manager-backup-recovery.adoc[leveloffset=+2]
```

Comment:
@ogradyp Why are these includes commented out instead of removed, and the files that they reference aren't removed as well? AFAICT, they aren't used anywhere else.

Reply:
Thanks @tradej for your feedback. Yes, they will be removed. This was an initial build to make sure it built ok. I searched and the files are not used / referenced anywhere else.